### PR TITLE
fix(cli): implement `--prefix` argument

### PR DESCRIPTION
**Pull request by luckasRanarison**
- Fixes [MET-738](https://linear.app/metatypedev/issue/MET-738/implement-prefix-in-meta-serialize).

#### Migration notes

---

- [ ] The change comes with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

File: src/typegraph/deno/src/envs/cli.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export function loadCliEnv(): CliEnv | null {
   for (const key of optionalCliEnvs) {
     const name = `MCLI_${key.toLocaleUpperCase()}`;
     const envValue = env[name];
-    if (envValue != null) {
+    if (envValue) {
```
Comment (Natoandro) [inline src/typegraph/deno/src/envs/cli.ts:81]: Why? :stuck_out_tongue:

**Review by Natoandro — DISMISSED** [review]

File: src/typegraph/deno/src/envs/cli.ts (hunk 1 above)
Comment (luckasRanarison) [reply src/typegraph/deno/src/envs/cli.ts:81]: Because optional args are just **empty** strings

**Review by Natoandro — APPROVED** [review]

File: src/meta-cli/src/deploy/actors/task/command.rs [hunk 2]
```diff
@@ -100,6 +100,8 @@ impl CommandContext {
             path,
         } = self;
 
+        let optional_envs = [("MCLI_PREFIX", shared_config.prefix.clone())];
+
```
Comment (j03-dev) [inline src/meta-cli/src/deploy/actors/task/command.rs:104]: It's only containing one element, add all `envs` here or just, use `env`.

**Review by j03-dev — APPROVED** [review]

File: src/meta-cli/src/deploy/actors/task/command.rs (hunk 2 above)
Comment (luckasRanarison) [reply src/meta-cli/src/deploy/actors/task/command.rs:104]: It's just for readability and future use cases.

**Pull request merged**